apollo-portal/src/main/java/com/ctrip/framework/apollo/openapi/service/ConsumerService.java

```diff
@@ -249,6 +258,17 @@ private List<Boolean> isAllowCreateApplication(List<Long> consumerIdList) {
     return list;
   }
 
+  private List<Integer> getRateLimit(List<Long> consumerIdList) {
+    List<Integer> list = new ArrayList<>(consumerIdList.size());
+    for (Long consumerId : consumerIdList) {
+      ConsumerToken consumerToken = consumerTokenRepository.findByConsumerId(consumerId);
```

Comment:
Adding a method like `List<ConsumerToken> findByConsumerIdIn(List<Long> consumerIds)` in ConsumerTokenRepository could improve efficiency.